Example/Tests/GINICameraViewControllerTests.swift

```diff
@@ -29,11 +32,60 @@ class CameraViewControllerTests: XCTestCase {
         GiniConfiguration.sharedConfiguration.onboardingShowAtLaunch = false
         GiniConfiguration.sharedConfiguration.onboardingShowAtFirstLaunch = false
         
-        vc = CameraViewController(successBlock: { _ in }, failureBlock: { _ in })
         _ = vc.view
         
         XCTAssertFalse(vc.captureButton.isEnabled, "capture button should be disaled when tooltip is shown")
         
     }
+    
+    func testReviewButtonBackground() {
+        _ = vc.view
```

Comment:
Because there was the issue with the GiniConfiguration singleton on the tests above. I could either move it to the setUp method or move the vc initialization to the tests which need a fresh initialization.